calico/openstack/mech_calico.py

```diff
@@ -703,17 +720,13 @@ def resync_endpoints(self, context):
             try:
                 etcd_data = json.loads(endpoint.data)
             except (ValueError, TypeError):
-                # If the JSON data is bad, just ignore it. We can't blow up
-                # here because that will trigger a new resync on another node
-                # that will blow *it* up as well. Just tolerate it.
+                # If the JSON data is bad, we need to fix it up. Set a value
+                # that is impossible for Neutron to be returning: nothing at
```

Comment:
There is nothing particularly special about the empty list. =P Switching to `None`.